### DISCUSSION TITLE
Proxy Support?

### DISCUSSION THREAD

**Discussion by bmaster** [microsoft/powerplatform-build-tools#224] [Q&A] [answered]
I am using DevOps Server (on-premise) and have proxy settings configured in my build agent.  The Power Platform steps do not seem to be respecting the proxy settings.  Does the tool support proxy settings for those of us trying to utilize this in a corporate environment?

**Comment by petrochuk** [comment, marked as answer]
Our tooling doesn't do anything special related to proxies. At the end we use HttpClient which has some [proxy support](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.defaultproxy?view=net-7.0)
